packages/data-format/test/data/example-meta-unknown.json

```diff
@@ -1,78 +1,80 @@
-{
-  "meta": {
-    "format": "rnf_invoice",
-    "version": "999999.0.0"
-  },
-
-  "creationDate": "2018-01-01T18:25:43.511Z",
-  "invoiceNumber": "123456789",
-  "purchaseOrderId": "987654321",
-  "note": "this is an example of invoice",
-  "terms": "there is no specific terms",
-
-  "sellerInfo": {
-    "email": "jean.valjean@miserables.fr",
-    "firstName": "Jean",
-    "lastName": "Valjean",
-    "phone": "+33606060606",
-    "address": {
-      "locality": "Paris",
-      "postal-code": "F-75002",
-      "street-address": "38 avenue de l'Opera",
-      "country-name": "France"
-    },
-    "miscellaneous": {
-      "aliases": ["Ultime Fauchelevent", "Urbain Fabre", "Prisoner 24601", "Prisoner 9430"]
-    }
-  },
-
-  "buyerInfo": {
-    "email": "javertlimited@detective.com",
-    "businessName": "Javert Limited",
-    "phone": "+16501123456",
-    "address": {
-      "locality": "Seattle",
-      "region": "WA",
-      "country-name": "United-State",
-      "postal-code": "98052",
-      "street-address": "20341 Whitworth Institute 405 N. Whitworth"
-    },
-    "miscellaneous": {
-      "Occupation": ["Prison guard", "Police inspector", "Detective"]
-    }
-  },
-
-  "invoiceItems": [
-    {
-      "name": "Candlestick",
-      "reference": "cs666",
-      "quantity": 2,
-      "unitPrice": 0.01,
-      "discount": 0.002,
-      "taxPercent": 16.9,
-      "deliveryDate": "2019-01-01T18:25:43.511Z"
-    },
-    {
-      "name": "handcuff",
-      "reference": "hc99",
-      "quantity": 1,
-      "unitPrice": 0.001,
-      "taxPercent": 5.5,
-      "deliveryDate": "2019-01-01T18:25:43.511Z"
-    }
-  ],
-
-  "paymentTerms": {
-    "dueDate": "2019-06-01T18:25:43.511Z",
-    "lateFeesPercent": 10,
-    "lateFeesFix": 0.01,
-    "miscellaneous": {
-      "note": "payment before chrismas"
-    }
-  },
-
-  "miscellaneous": {
-    "manufacturerCompany": "Victor Hugo & Co.",
-    "deliveryCompany": "Gavroche Express"
-  }
-}
+{

```

Comment:
why are we changing everything on these files?

Reply:
It's a formatting issue - taking a look to see why its happened